src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -4,6 +4,9 @@ public abstract class Machine implements Workable {
     private String name;
     private String color;
 
+    public Machine() {
```

Comment:
```suggestion
    public Machine(String name, String color) {
          this.name = name;
          this.color = color;
```
and use this constructor inside other constructors